scipy/sparse/tests/test_base.py

```diff
@@ -3544,6 +3527,35 @@ def test_constructor6(self):
         assert_array_equal(csr.shape, (3,6))
         assert_(np.issubdtype(csr.dtype, np.signedinteger))
 
+    def test_constructor_smallcol(self):
+        # int64 indices not required
+        data = arange(6) + 1
+        col = array([1, 2, 1, 0, 0, 2], dtype='int64')
+        ptr = array([0, 2, 4, 6], dtype='int64')
```

Comment:
Use `np.int64` instead of the string version? (Here and elsewhere)

Reply:
Done.